lib/app/ctx.js

```diff
@@ -220,10 +220,16 @@ Box.prototype.unwrap = function (data, content) {
             if (msg.type.substr(3) === '_CRYPT' || msg.type.substr(3) === '_SIGN') {
                 data = msg.contents;
             }
-
             if (msg.type.substr(0, 3) === 'QLB' && tr.docs.length > 0) {
                 content = tr.docs.shift().contents;
             }
+            if (msg.type === "PACKED_XML") {
```

Comment:
Recursion here gives an easy way to split one input file, that's the reason I used it here.
I can try to move this processing to transport.js, but then the recursion will be there, and transport_decode will return an array of objects which I'll need to handle here in a special way (do not try to interpret as ASN.1 data).